src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/Models/PendingPayment.cs

```diff
@@ -18,5 +18,6 @@ public partial class PendingPayment
         public short? PeriodNumber { get; set; }
         public short? PaymentYear { get; set; }
         public long AccountLegalEntityId { get; set; }
+        public short? EarningType { get; set; }
```

Comment:
Can't we use an enum?

Reply:
Stored as a string in the db, same way we do with application status.

Reply:
We can, the ticket was a little vague on what should actually be stored so had to guess :)